### ISSUE TITLE
Outdated content View sections grouping is inaccessible

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#11956]
## User Story or Problem Statement
The grouping by section makes for an ugly display.  It is also a nightmare for 508 because we can't use table captions AND it just stacks table on top of table.  CC: @laflannery 
![image](https://user-images.githubusercontent.com/5752113/207780002-3780182e-2aa7-4216-a277-832eb7fecae4.png)

The functionality doesn't gain us anything since we have the ability to filter by section or sort by section.
Suggestion: should consider removing the section grouping.

## Acceptance Criteria
- [ ] Table has a meaningful caption of "___"
- [ ] Table is not grouped by sections
- [ ] Sorting by section is still supported
- [ ] Filtering by section is still supported

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@swirt I think I'm confused by the first issue where you say "we can't use table captions" - each table does have a table caption currently. Whether or not we should have multiple tables we can discuss also but I do want to make sure we are taking about the same thing for this first point.

**Comment by EWashb** [comment]
@swirtSJW did you have time to chat about this one?

**Comment by laflannery** [comment]
The only outstanding issue here is that this view doesn't have a caption, but that's the case with many of our content views so it's a larger issue. I will do an audit of tables/views at a later date to have these updates with captions
